python/dlr/api.py

```diff
@@ -43,6 +43,15 @@ def _find_model_file(model_path, ext):
         return model_files[0]
     return None
 
+def _find_saved_model(model_path):
```

Comment:
1. Lets add tf2 to the method name
2. Looks like `_find_model_file` is not used anymore

Reply:
1. done
2. removed `_find_model_file`